torch/csrc/jit/export.cpp

```diff
@@ -544,7 +611,9 @@ GraphEncoder::GraphEncoder(
 void GraphEncoder::EncodeTensor(
     onnx::TensorProto* tensor_proto,
     const at::Tensor& tensor,
-    const c10::optional<std::string> external_ref) {
+    const c10::optional<std::string> external_ref,
+    const bool use_external_data_format,
+    const std::string& onnx_file_path) {
```

Comment:
Nit: onnx_param_file_path or onnx_weights_file_path?

Reply:
Actually, at this point this is the file path for the ONNX model file itself. We derive the param/weight file name from this (variable named `fullFilePath`). So I feel that this name may be appropriate. Let me know if that's OK, otherwise I can update it.